kolibri/core/assets/src/views/ExamReport/PageStatus.vue

```diff
@@ -107,6 +141,42 @@
           return this.$tr('notStartedLabel');
         }
       },
+      retakeEnabled() {
+        // The Retake button is enabled only for leaners,
+        // the threshold should be kept in a proper conf and not here (TODO)
+        var isScoreBelowThreshold =
+          this.questions.reduce((a, q) => a + q.correct, 0) / this.questions.length || 0 < 90;
```

Comment:
two things here -- 

this line is a bit difficult for the human eye to parse -- I would suggest breaking it up or doing the reduce as its own thing.  I know they aren't necessary (the order of operations is known to the computer)  but parentheses might help for grouping with the human eye and avoid the need for external docs
 ( I know that you didn't add much to the expression -- It was original terse)